sql-plugin/src/main/scala/com/nvidia/spark/rapids/DateUtils.scala

```diff
@@ -211,4 +213,55 @@ object DateUtils {
   }
 
   case class TimestampFormatConversionException(reason: String) extends Exception
+
+  def tagAndGetCudfFormat(
+      meta: RapidsMeta[_, _, _],
+      sparkFormat: String,
+      parseString: Boolean): String = {
+    var strfFormat = ""
```

Comment:
nit: I would feel better if this was set like.

```
var strfFormat: String
```

Possibly even setting it to null first. Just because an empty format might actually produce results and be harder to debug compered to a null or something the compiler checks is being set.